interpreter/interpreter.cabal

```diff
@@ -0,0 +1,39 @@
+cabal-version: >=1.10
+name: interpreter
+version: 0.1.0.0
+license: BSD3
```

Comment:
We're using Apache 2 now. See the other projects, and you need to copy in the `LICENSE`, `NOTICE`, and remove the `copyright` line here.